packages/axelar-std/src/address.rs

```diff
@@ -13,4 +15,12 @@ impl AddressExt for Address {
     fn zero(env: &Env) -> Address {
         Self::from_string(&String::from_str(env, ZERO_ADDRESS))
     }
+
+    // Converts Stellar address (soroban_sdk::Address) to soroban_sdk::Bytes
+    fn to_bytes(&self, env: &Env) -> Bytes {
+        let address_str = self.to_string();
+        let mut str_bytes = [0u8; STELLAR_ADDRESS_LEN];
+        address_str.copy_into_slice(&mut str_bytes);
+        Bytes::from_slice(env, &str_bytes)
+    }
 }
```

Comment:
add unit test to convert address to bytes and back to ensure it works. you can try with zero address and random addresses

Reply:
try different address types that are supported
https://docs.rs/soroban-sdk/latest/soroban_sdk/struct.Address.html#method.from_string_bytes